tests/data/many-types/input_stub.cpp

```diff
@@ -420,29 +420,32 @@ void org::gdbus::codegen::glibmm::Test::on_method_call(
         parameters.get_child(base_Param1, 0);
         std::map<Glib::ustring,Glib::VariantBase> p_Param1 = base_Param1.get();
 
+        MethodInvocation method_invocation(invocation);
```

Comment:
maybe `methodInvocation` instead

Reply:
Can change this... are you sure it is the preferred style though? It seems to be pretty mixed...

Reply:
Thats what we use on cpp code, in python code `method_invocation` would be better choice.

Reply:
I see plenty of unders score everywhere in the C++ code, but camel case is the way we want to go?

Reply:
Changed it to camelCase.